app/vmalert/config.go

```diff
@@ -69,5 +69,10 @@ func parseFile(path string) ([]Group, error) {
 		Groups []Group `yaml:"groups"`
 	}{}
 	err = yaml.Unmarshal(data, &g)
+	for i := 0 ; i < len(g.Groups); i++ {
```

Comment:
Could you pls cover this case in config tests to verify that evaluation interval will be correct?

Reply:
Fixed